### PR TITLE
Juror-Draft-Court-Rulling wording changes

### PR DESCRIPTION
**Pull request by rperez89**
Built on top of #1 we need to merge that one first.

### CODE REVIEW

File: docs/metadata.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ type Disputable @entity {
 }
 ```
 
-### Aragon court handles two different types of disputes:
+### Aragon protocol handles two different types of disputes:
```
Comment (sohkai) [inline docs/metadata.md:39]:
```suggestion
### Aragon Protocol handles two different types of disputes:
```

File: src/components/AppLoader.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ function AppLoader({ children }) {
   const courtConfig = useCourtConfig()
 
   if (!courtConfig) {
-    return <SyncIndicator visible label="Loading court…" /> // TODO: Add better loader
+    return <SyncIndicator visible label="Loading protocol…" /> // TODO: Add better loader
```
Comment (sohkai) [inline src/components/AppLoader.js:9]:
```suggestion
    return <SyncIndicator visible label="Loading Protocol…" /> // TODO: Add better loader
```

File: src/types/dispute-status-types.js [hunk 3]
```diff
@@ -31,16 +31,16 @@ const stringMapping = {
   [Status.Closed]: 'Closed',
   [Phase.All]: 'All',
   [Phase.Evidence]: 'Adding arguments',
-  [Phase.JuryDrafting]: 'Jury drafting',
+  [Phase.JuryDrafting]: 'Guardians summon',
```
Comment (sohkai) [inline src/types/dispute-status-types.js:34]:
```suggestion
  [Phase.JuryDrafting]: 'Summon guardians',
```

Perhaps this sounds better haha

**Review by sohkai — APPROVED** [review]
Couple of small copy notes, thanks guys!

File: src/types/dispute-status-types.js (hunk 3 above)
Comment (rperez89) [reply src/types/dispute-status-types.js:34]: hahah ahh thanxs for your review, i was going to ask paty to review the summon wording i have lot of dubts, pretty weird word for me haha

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-# Aragon Court Dashboard <a href="https://aragon.org/"><img align="right" src="docs/assets/LogoAccent.svg" height="80px" /></a>
+# Aragon Protocol Dashboard <a href="https://aragon.org/"><img align="right" src="docs/assets/LogoAccent.svg" height="80px" /></a>
 
-### Aragon Court handles subjective disputes requiring judgment from human jurors.
+### Aragon Protocol handles subjective disputes requiring judgment from human jurors.
```
Comment (fabriziovigevani) [inline README.md:3]:
```suggestion
### Aragon Protocol handles subjective disputes requiring judgment from human guardians.
```

**Review by fabriziovigevani — COMMENTED** [review]
Awesome Ro! Some comments and suggestions.

Also, below is a list of lines where we missed some updates:

**Court => Protocol**

- `README.md`: line 10
- `CONFIGURATION.md`: line 21 and 49
- `ClockModule.js`: line 185 and 267

**Ruling => Decision**

- `AppealPanel.js`: Line 124

**Juror => Guardian**
- `DisputeTimeline.js`: Line 181

File: src/components/Disputes/actions/DisputeDraft.js [hunk 5]
```diff
@@ -24,12 +24,12 @@ function DisputeDraft({ disputeId, onDraft }) {
...
       >
         <Button type="submit" mode="strong" wide disabled={!wallet.account}>
-          Draft jury
+          Summon guardian
         </Button>
       </div>
       <Info>
         The evidence submission period is closed. <strong>Anyone</strong> can
-        now trigger the drafting of a jury and earn some rewards.
+        now trigger the summon of a guardian and earn some rewards.
```
[the remark this answers is not in the document]
Comment (rperez89) [reply src/components/Disputes/actions/DisputeDraft.js:32]: cc @dizzypaty

File: docs/CONFIGURATION.md [hunk 6]
```diff
@@ -46,7 +46,7 @@ To disable flagging of voided disputes.
 
 ### MOCK_DATA
 
-To start the court dashboard with mocked data.
+To start the Aragon Protocol with mocked data.
```
Comment (fabriziovigevani) [inline docs/CONFIGURATION.md:49]:
```suggestion
To start the Aragon Protocol dashboard with mocked data.
```

File: src/components/Disputes/panels/AppealPanel.js [hunk 7]
```diff
@@ -121,7 +121,7 @@ function AppealPanel({
     return false
   }
 
-  const actionLabel = confirm ? 'Confirm appeal' : 'Appeal ruling'
+  const actionLabel = confirm ? 'Confirm appeal' : 'Appeal decisions'
```
Comment (fabriziovigevani) [inline src/components/Disputes/panels/AppealPanel.js:124]:
```suggestion
  const actionLabel = confirm ? 'Confirm appeal' : 'Appeal decision'
```

**Review by fabriziovigevani — APPROVED** [review]
Two last comments and LGTM!

File: README.md [hunk 8]
```diff
@@ -1,21 +1,21 @@
...
-### Aragon Court handles subjective disputes requiring judgment from human jurors.
+### Aragon Protocol handles subjective disputes requiring judgment from human guardians.
 
 <a href="https://court.aragon.org/dashboard"><img src="docs/assets/screenshot.png" /></a>
 
 - 📚 Read the [User Guide](https://help.aragon.org/category/47-aragoncourt) first, if you have any questions as a user.
 - 📝 Please report any issues and feedback in the [Aragon discord channels](https://discord.gg/zcKs8H).
 - 🔧 For technical discussions, use this project's [issues](https://github.com/aragon/court-dashboard/issues) or join the technical conversation in our [Developer hangout](https://spectrum.chat/aragon/general-development) channel.
-- 📖 To learn more about contributing to the Court Dashboard, please check the [contributing guide](./CONTRIBUTING.md)
+- 📖 To learn more about contributing to the Aragon Protocol, please check the [contributing guide](./CONTRIBUTING.md)
```
Comment (sohkai) [inline README.md:10]:
```suggestion
- 📖 To learn more about contributing to the Aragon Protocol Dashboard, please check the [contributing guide](./CONTRIBUTING.md)
```

File: README.md [hunk 9]
```diff
@@ -29,9 +29,9 @@ In order to start the dashboard locally, you will need to setup a few processes
 
  _For a detailed guide on how to achieve this, see more instructions [here](https://github.com/aragon/court-subgraph)._
 
-You can also setup the [court services](https://github.com/aragonone/court-backend/tree/master/packages/services), which include some automation for auto-revealing commitments, penalty settlements and heartbeats. In order for auto-reveals to work, you will need also to setup the [court server](https://github.com/aragonone/court-backend/tree/master/packages/server).
+You can also setup the [protocol services](https://github.com/aragonone/court-backend/tree/master/packages/services), which include some automation for auto-revealing commitments, penalty settlements and heartbeats. In order for auto-reveals to work, you will need also to setup the [protocol server](https://github.com/aragonone/court-backend/tree/master/packages/server).
```
Comment (sohkai) [inline README.md:32]:
```suggestion
You can also setup the [protocol services](https://github.com/aragon/protocol-backend/tree/master/packages/services), which include some automation for auto-revealing commitments, penalty settlements and heartbeats. In order for auto-reveals to work, you will need also to setup the [protocol server](https://github.com/aragon/protocol-backend/tree/master/packages/server).
```

File: README.md [hunk 10]
```diff
@@ -29,9 +29,9 @@ In order to start the dashboard locally, you will need to setup a few processes
...
 
-To start populating the court with disputes, you can use the [CLI tool](https://github.com/aragonone/court-backend/tree/master/packages/cli).
+To start populating the protocol with disputes, you can use the [CLI tool](https://github.com/aragonone/court-backend/tree/master/packages/cli).
```
Comment (sohkai) [inline README.md:34]:
```suggestion
To start populating the protocol with disputes, you can use the [CLI tool](https://github.com/aragon/protocol-backend/tree/master/packages/cli).
```

File: README.md [hunk 11]
```diff
@@ -43,7 +43,7 @@ For other details about contributing here, more information is available in the
 
 #### Issues
 
-If you come across an issue with the Court Dashboard, do a search in the [Issues](https://github.com/aragon/court-dashboard/issues?q=is%3Aissue+is%3Aopen) tab of this repo to make sure it hasn't been reported before. Follow these steps to help us prevent duplicate issues and unnecessary notifications going to the many people watching this repo:
+If you come across an issue with the Protocols Dashboard, do a search in the [Issues](https://github.com/aragon/protocol-dashboard/issues?q=is%3Aissue+is%3Aopen) tab of this repo to make sure it hasn't been reported before. Follow these steps to help us prevent duplicate issues and unnecessary notifications going to the many people watching this repo:
```
Comment (sohkai) [inline README.md:46]:
```suggestion
If you come across an issue with the Protocol Dashboard, do a search in the [Issues](https://github.com/aragon/protocol-dashboard/issues?q=is%3Aissue+is%3Aopen) tab of this repo to make sure it hasn't been reported before. Follow these steps to help us prevent duplicate issues and unnecessary notifications going to the many people watching this repo:
```

File: README.md [hunk 12]
```diff
@@ -1,21 +1,21 @@
-# Aragon Court Dashboard <a href="https://aragon.org/"><img align="right" src="docs/assets/LogoAccent.svg" height="80px" /></a>
+# Aragon Protocol Dashboard <a href="https://aragon.org/"><img align="right" src="docs/assets/LogoAccent.svg" height="80px" /></a>
 
-### Aragon Court handles subjective disputes requiring judgment from human jurors.
+### Aragon Protocol handles subjective disputes requiring judgment from human guardians.
 
 <a href="https://court.aragon.org/dashboard"><img src="docs/assets/screenshot.png" /></a>
 
 - 📚 Read the [User Guide](https://help.aragon.org/category/47-aragoncourt) first, if you have any questions as a user.
 - 📝 Please report any issues and feedback in the [Aragon discord channels](https://discord.gg/zcKs8H).
```
Comment (sohkai) [inline README.md:8]:
```suggestion
- 📝 Please report any issues and feedback in the [support channel on Discord](https://discord.gg/ZncWSEGbSJ).
```

File: README.md [hunk 13]
```diff
@@ -1,21 +1,21 @@
...
 - 🔧 For technical discussions, use this project's [issues](https://github.com/aragon/court-dashboard/issues) or join the technical conversation in our [Developer hangout](https://spectrum.chat/aragon/general-development) channel.
```
Comment (sohkai) [inline README.md:9]:
```suggestion
- 🔧 For technical discussions, use this project's [issues](https://github.com/aragon/court-dashboard/issues) or join the technical conversation in join the technical conversation in the [hacker hangout channel on Discord](https://discord.gg/XrhUzEy7jf).
```

**Review by sohkai — APPROVED** [review]
Doing a quick search for `court`, `juror`, and `anj` still ends up with a lot of results haha.

Should we tackle these renamings after this and #3 are merged before progressing any further?

**Comment by rperez89** [comment]
> Doing a quick search for `court`, `juror`, and `anj` still ends up with a lot of results in the codebase haha.
> 
> Should we tackle these renamings after this and #3 are merged before progressing any further?

oh damn probably i was searching with some blank space or something because those ones didn't show up. Thxs

**Comment by rperez89** [comment]
> Doing a quick search for `court`, `juror`, and `anj` still ends up with a lot of results in the codebase haha.
> 
> Should we tackle these renamings after this and #3 are merged before progressing any further?

we didn't change the variables names.... just in case hahha

**Comment by sohkai** [comment]
> we didn't change the variables names.... just in case hahha

Haha I noticed, but I was thinking that we might as well go through and do them since it should be relatively effortless with some `sed` or a find and replace all type of operation. Otherwise the codebase doesn't really make sense in the eyes of the Aragon Protocol.

Aside from that, I did still notice some copy using those terms that were missed, but related to above, it's easier to ensure we've eradicated all instances if we change the variable names too ;).

**Comment by rperez89** [comment]
> > we didn't change the variables names.... just in case hahha
> 
> Haha I noticed, but I was thinking that we might as well go through and do them since it should be relatively effortless with some `sed` or a find and replace all type of operation. Otherwise the codebase doesn't really make sense in the eyes of the Aragon Protocol.
> 
> Aside from that, I did still notice some copy using those terms that were missed, but related to above, it's easier to ensure we've eradicated all instances if we change the variable names too ;).

yes we were talking with facu about that , that we might want do that at the end once we finish the other tasks, should i also change ANJ  wording now?

File: README.md (hunk 10 above)
Comment (rperez89) [reply README.md:34]: ah dind' change the links because just in case the repo dind't exist, did you guys already moved to that repo?

**Comment by sohkai** [comment]
Got it, let's not worry about it now then! Functionality first, then we can go through and replace with our editors :).

File: README.md (hunk 10 above)
Comment (sohkai) [reply README.md]: Yes, it's already up!

**Comment by rperez89** [comment]
> Got it, let's not worry about it now then! Functionality first, then we can go through and replace with our editors :).

cool let's merge this pr if you are ok and will continue with the others issues created regarding the anj

**Pull request merged**